lib/cli/src/frameworks/web-components/js/Page.js

```diff
@@ -1,6 +1,5 @@
 import { html } from 'lit-html';
 import { Header } from './Header';
-import './page.css';
```

Comment:
@43081j I'f I'm right, you removed it, is it automatically importing css ?

Reply:
I removed it originally because when i changed to typescript, the import couldn't be found. Where does this CSS live?

Really I don't think it should exist for the web components examples anyway. 

Each component has its own scoped styles so it wouldn't make any sense trying to import a css file here. As you would be applying css to the document which can't even penetrate the shadow dom boundaries of these components. So I don't think it's a use case anymore unlike react for example

Reply:
In real world you would import an object like:

```
import {styles} from './pageStyles.js';
```

Which is (in lit):

```
export const styles = css`
  :host { display: block; }
`;
```

Styles are always scoped as per the spec so most libraries will do it this way. Or they will produce something like this at build time

Reply:
Styles are shared between all frameworks and are added on user side. What you propose is probably how it should be done but we also need to find the balance with the current architecture. using css files is the best option

Reply:
The problem is, using CSS files just wouldn't have any effect with a web component.. so i'd argue we should just not have a css import in the web component examples to avoid confusion.

It isn't a pattern you will see, as the most you could do is style everything _outside_ the component in that stylesheet. A web component would normally have its own shadow root, so those outside styles would have no effect on it (other than variables and host styles).

It isn't really a use case when dealing with web components because of this so i just want to make sure we're not including it just because non-web-component frameworks do it.